Tasks/UserInterface/01a_List.md

```diff
@@ -0,0 +1,156 @@
+[!!Manage the tasks](../Operation/02_ManageTasks.md)
+[!!User Interface DataHub](../../DataHub/UserInterface/00_UserInterface.md)
+
+# List (Tasks)
+
+*Tasks > Tasks > Tab LIST*
+
+![Tasks](../../Assets/Screenshots/Tasks/Tasks/ListTasks.png "[Tasks]")
+
+**Tasks**
+
+You can filter the tasks displayed by different criteria using the *Assignees*, *Events*, and *Status* drop-down lists. You can also combine the filters to narrow down the listed tasks. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Assignees*  
+    Click the drop-down list to filter the list of tasks by assignee. The following options are available:  
+    - **All**  
+        Select this option to display all the tasks, regardless of the assignee.
+    - **Assigned directly to me**  
+        Select this option to display the tasks assigned specifically to you. 
+    - **Assigned to me via groups**  
+        Select this option to display the tasks assigned to a user group which you are part of.
+    - **Assigned to me directly and via groups**  
+        Select this option to display the tasks assigned to you, regardless of whether they are assigned to you specifically or you are part of a user group to which the task is assigned.
+    - **Unassigned**  
+        Select this option to display the tasks that are not assigned to any user or group.
+
+[comment]: <> (Es sollte Assigned to me directly ODER via groups sein! RS mit Modulexperten) 
+
+- *Events*  
+    Click the drop-down list to filter the list of tasks by event. All events created in the *Events* menu entry are displayed.
+
+- *Status*  
+    Click the drop-down list to filter the list of tasks by status. The following options are available:  
+    - **Open tasks**  
+        Select this option to display only the open, that is, pending, tasks.
+    - **Open and closed tasks**  
+        Select this option to display all tasks, both open (pending) and closed (done).
+
+The following functions are available for the editing toolbar: 
+
+- [MARK DONE]  
+    Click this button to mark a task as done. This button is displayed if the checkbox of at least one open task, with status ![Cross](../../Assets/Icons/Cross05.png "[Cross]"), is selected.
+
+- [REOPEN]  
+    Click this button to reopen a task. This button is displayed if the checkbox of at least one closed task, with status ![Check](../../Assets/Icons/Check02.png "[Check]"), is selected.
+
+- [ASSIGN TO ME]  
+    Click this button to assign a task to yourself. This button is displayed if the checkbox of at least one task is selected.
+
+- *Unassigned*  
+    Click the drop-down list to select a user to assign him/her a task. This button is displayed if the checkbox of at least one task is selected. The list displays all users that are registered in your system instance. 
+
+    > [Info] You can assign an unassigned task as well as reassign a task that is currently assigned to another user.
```

Comment:
Gibt es da eigentlich bestimmte Berechtigungen (->Prerequisites)? Ich kann mir vorstellen, dass das nicht jeder darf. Siehe Manage the tasks.

Reply:
Extra gefragt. Nee, zur Zeit keine Berechtigungen/Einschränkungen.